stdlib/prelude.an

```diff
@@ -674,23 +674,24 @@ next_line (f: InFile) : string =
     array_insert (@cstr) len '\0'
     string (@cstr) len
 
-
-
-/// Print a message then return user input
-input msg : string =
-    printne msg
-
+// Read a single line from stdin
+read_line : string =
```

Comment:
Looks like you made a variable here instead of a function, you'll have to add a parameter like so: `read_line () : string = ...`

Reply:
Oops! Guess that means it would have only worked exactly once. Fixed now I hope.